core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1074,6 +1074,8 @@ private Name() {} // prevent instantiation
         "alluxio\\.master\\.mount\\.table\\.(\\w+)\\.shared"),
     MASTER_MOUNT_TABLE_ENTRY_UFS("alluxio.master.mount.table.%s.ufs",
```

Comment:
would it make sense to remove `ENTRY` from the template names? (the key template does not contain it)